mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/BoostingStrategy.scala

```diff
@@ -34,6 +34,12 @@ import org.apache.spark.mllib.tree.loss.{LogLoss, SquaredError, Loss}
  *                      weak hypotheses used in the final model.
  * @param learningRate Learning rate for shrinking the contribution of each estimator. The
  *                     learning rate should be between in the interval (0, 1]
+ * @param validationTol Useful when runWithValidation is used. If the error rate between two
+                        iterations is lesser than the validationTol, then stop. If run
```

Comment:
"lesser" --> "less"

Also, use double brackets to add link to "run" to be clearer.